src/NzbDrone.Core.Test/DecisionEngineTests/RssSync/DelaySpecificationFixture.cs

```diff
@@ -104,10 +104,22 @@ public void should_be_true_when_profile_does_not_have_a_delay()
         }
 
         [Test]
-        public void should_be_true_when_quality_is_last_allowed_in_profile()
+        public void should_be_false_when_quality_is_last_allowed_in_profile_and_bypass_disabled()
         {
             _remoteMovie.ParsedMovieInfo.Quality = new QualityModel(Quality.Bluray720p);
 
+            _remoteMovie.Release.PublishDate = DateTime.UtcNow;
+
+            Subject.IsSatisfiedBy(_remoteMovie, null).Accepted.Should().BeFalse();
```

Comment:
This was my fix to the test failing - Taloth never got back to me but as the test was name "should be false" I assumed this should be false and not true as it is in the Sonarr Cherry-Pick :)